code/oled_block/oled.py

```diff
@@ -0,0 +1,53 @@
+import Adafruit_GPIO.SPI as SPI
+import Adafruit_SSD1306
+import time
+
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+
+oled_buffer_path = 'oled_buffer.txt' # Path to oled buffer
+line_width = 20 # Number of characters in one line
```

Comment:
I believe these couple variables are actually constants, in Python we use `CAPITAL_LETTERS` for constants

Reply:
Oke bruh